packages/date-time/src/components/DatePicker/DatePicker.styles.ts

```diff
@@ -52,8 +52,12 @@ export const styles = (props: IDatePickerStyleProps): IDatePickerStyles => {
     callout: [classNames.callout],
     icon: [
       DatePickerEvent,
-      !label && [classNames.withoutLabel, { top: '7px' }],
-      label && [classNames.withLabel, { bottom: '5px' }],
+      {
```

Comment:
Why not add these directly to DatePickerEvent?

Reply:
I had thought, from the name, that the const was used elsewhere but it doesn't seem to be. I should rename it and add these then. DatePickerEvent is a misleading name for its usage.